packages/syft/src/syft/core/node/domain_client.py

```diff
@@ -408,15 +403,6 @@ def reset(self) -> None:
         if response == "y":
             response = self.routes[0].connection.reset()  # type: ignore
 
-    def search(self, query: List, pandas: bool = False) -> Any:
```

Comment:
Q: Do we not use network search message anymore?

Reply:
We removed only from the Domain Client. But it's still in NetworkClient API
<img width="857" alt="network_client_search" src="https://user-images.githubusercontent.com/26658472/201928957-5d838c2e-d29e-4127-915b-13dd82ff2573.png">

Reply:
Ahh ok